### ISSUE TITLE
Add ReplayID to Scoreboard

### ISSUE THREAD

**Issue by MrRowey** [FAForever/fa#3934]
I know by hoving over the map name the ReplayID is located there but if someone is watching a cast tournament video coverage being able to see the exact replay would be beneficial as this is already a thing that's a part of the supreme scoreboard mod. plus would make my life easier as Promo Team when uploading tournament coverage and knowing the exact replay for that game. 
A clear and concise description of what the feature is. 
![image](https://user-images.githubusercontent.com/20344151/172587640-8600f6a2-c466-4c38-80b5-72ed2ba667f3.png)

**Comment by Hdt80bro** [comment]
Something like this?
![image](https://user-images.githubusercontent.com/37224614/172732907-e1ba64be-415c-4dd7-8566-4b3ac33801a6.png)
Or are you talking about a more drastic UI rewrite.

**Comment by MrRowey** [comment]
@Hdt80bro Yes this is what I'm looking for. but i know what what jip wants to do by redoing the scoreboard

**Comment by Hdt80bro** [comment]
Yeah, I do too. I was just trying to quickly rearrange the text and anchor the id to the right of the score panel. But it turns out that there's several things competing for that corner. For instance, if the observerLine is showing, then the resource switcher moves up (for some reason) creating void space that needs to be filled. Unless the ranked icon is showing, then it should anchor to that...
![image](https://user-images.githubusercontent.com/37224614/172734799-80a71234-d7d0-49cc-94ab-94dc27a303db.png)
You can make it work, with effort. The only thing I haven't figured out is how to slide the Replay ID back over when you die and the observerLine shows
